lessons/operator/04/README.md

```diff
@@ -280,33 +163,153 @@ If so, you're ready to proceed to the next step!
      - has_metrics
      mutator: ""
    ```
+   
+   > **Understanding the YAML:**
+   > - The asset identifier `sensu/sensu-influxdb-handler:3.5.0` instructs the backend to download the handler executable from [Bonsai]. 
+   > - The built-in filter [`has_metrics`][has_metrics_docs] is used to ensure that only events with metrics are processed by the handler.
+   > - The handler is configured using environment variables and secrets available to the Sensu backend.
+   
 
-1. **Create the Handler using the `sensuctl create -f` command.**
+1. **Create the Handler Using the `sensuctl create` command.**
 
    ```shell
    sensuctl create -f influxdb.yaml
    ```
 
-   Verify that your handler was created:
+1. **Verify that the handler was created.**
 
    ```shell
    sensuctl handler list
    ```
 
-**NEXT:** Do you see the `influxdb` handler in the output?
-If so, you're ready to move on to the next step!
+   Do you see the `influxdb` handler in the output?
+   If so, you've successfully created the handler!
+
+## Discussion
+
+In this lesson, we've only scratched the surface of what handlers can do.
+You learned how to create a handler using YAML files, use handler templating to format event data, use built-in filters, use handlers published to Bonsai, and view a list of handler configurations.
+
+### Where Do Handlers Run?
+
+Handlers are part of the [process stage][process_stage_docs] of the [observability pipeline][observability_pipeline_docs]. That means all of this happens on the Sensu backend, running the same place where agents and checks send their observability data. 
+
+### Monitoring as Code and Sensu's API-based Architecture
+
+Because Sensu is [API-based][sensu_api_docs], we were able to create the handlers remotely, using `sensuctl` to push the desired configuration to the API.
+We used a [Monitoring as Code][monitoring_as_code_blog_post] workflow, storing the handler configurations in local YAML files. 
```

Comment:
"storing" or "creating"? :thinking:

Reply:
I think "authoring" might be better than either. Edited to reflect that.